app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1602,44 +1602,53 @@ table li .account_details {
   margin-bottom: 50px;
 }
 
-.row-items {
-  max-width: 1000px;
+// start non-admin items menu
+.items table {
+  background-color: #303030;
+  margin-top: 50px;
+  border: 1px solid #303030;
 }
 
-.items {
-  border: 1px solid red;
-  margin: 10px auto;
+.items td {
+  background-color: #303030;
+  color: white;
+  font-family: 'Cardinal', Arial, sans-serif;
+  text-align: center;
+  width: 275px;
+  // border-left: 1px solid #202020;
 }
 
-.item-image {
-  border: 1px solid red;
-  text-align: center;
+.item-title td {
+  font-size: 30px;
+  height: 50px;
 }
 
-.item-show-image {
-  border: 1px solid red;
-  text-align: center;
+.item-title a {
+  color: white;
 }
 
-.item-title {
-  border: 1px solid red;
-  text-align: center;
-  margin-top: 50px;
-  font-family: 'Cardinal', Arial, sans-serif;
-  font-size: 58px;
+.item-price td{
+  font-size: 23px;
+  height: 40px;
 }
 
-.item-title a {
-  color: white;
-  font-size: 24px;
+.item-description p{
+  height: 65px;
+  font-size: 23px;
+}
+
+.add-to-cart td{
+  border: 5px outset gray;
 }
 
-.item-price {
+.add-to-cart a{
+  font-family: 'Cardinal', Arial, sans-serif;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.